README.md

```diff
@@ -1,2 +1,27 @@
 # BridgeServer2-infra
-CloudFormation templates for BridgeServer2
+Install and configure the AWS Infrastructure for the BridgeServer 2.0 application.
+
+
+## Initialize a new stack
+1. Deploy the infrastructure to AWS. Run `sceptre launch-stack develop bridgeserver2`
+
+The above should create the infrastructure required for BridgeServer2, it does not actually deploy the app it only installs and configures the required AWS infrastructure for it. The AWS appilcation health check is enabled therefore the EB environment will start in an error state since the app has not been deployed. The environment status should turn green once the app is successfully deployed.
```

Comment:
Since we are changing from java to tomcat solution stack the last two sentence can probably be deleted.  I believe tomcat is packaged with an example app that will run on port 80 which means that app health should be green when deploying this infra.